samples/highcharts/annotations-advanced/controllable/demo.js

```diff
@@ -201,6 +201,106 @@ var chart = Highcharts.chart('container', {
                     }
                 }
             }]
+        }, {
+            type: 'ellipse',
```

Comment:
The ellipse in this demo is rendered out of the plot area. Could you correct the coordinates?